test/react_test_utils_test.dart

```diff
@@ -0,0 +1,277 @@
+import 'dart:js';
+
+import 'package:unittest/unittest.dart';
+import 'package:unittest/html_config.dart';
+import 'package:react/react.dart';
+import 'package:react/react_client.dart' as reactClient;
+import 'package:react/react_test_utils.dart';
+
+import 'test_components.dart';
+
+
+void main() {
+
+  useHtmlConfiguration();
+  reactClient.setClientConfiguration();
+
+  var component;
+  var domNode;
+  var _React = context['React'];
+  var _Object = context['Object'];
+
+  // This is intended only to make a simple representation of react elements
+  // as JS objects for testings purposes.
+  JsObject createReactElement(
+      String componentType, Map properties, List childrenComponents) {
+    var proto = new JsObject(_Object);
+    proto['render'] = () => null;
+    proto['setState'] = () => null;
+
+    var props = new JsObject(_Object);
+    properties.forEach((k, v) => props[k] = v);
```

Comment:
Is this what `JsObject.jsify(object)` does?  https://api.dartlang.org/apidocs/channels/be/dartdoc-viewer/dart:js.JsObject#id_JsObject-jsify